test/datastore/normalize_test.rb

```diff
@@ -102,13 +105,20 @@ class Datastore
                 end
                 it "raises if a potential followup stream has an non-matching logical time range" do
                     create_logfile "file0.1.log" do
-                        create_logfile_stream "stream0", metadata: Hash["rock_task_name" => "task0", "rock_task_object_name" => "port"]
+                        create_logfile_stream(
+                            "stream0",
+                            metadata: {
+                                "rock_task_name" => "task0",
+                                "rock_task_object_name" => "port"
+                            }
+                        )
                         write_logfile_sample base_time + 3, base_time + 10, 3
                     end
-                    capture_io do
-                        assert_raises(Normalize::InvalidFollowupStream) do
-                            normalize.normalize([logfile_pathname("file0.0.log"), logfile_pathname("file0.1.log")])
-                        end
+                    assert_raises(Normalize::InvalidFollowupStream) do
```

Comment:
why isnt it necessary to capture_io anymore?

Reply:
Mistake, was for debug

Reply:
Ping. Fixed.